data-load/user-load.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+#
+# Script to load a batch of new users into OAuth for new service access.
+#
+# Parameters:
+#       1. ENV - [t3|t2|preprod|prod]
```

Comment:
won't work for t2 out of the box as the `HOST` will be incorrect

Reply:
Ok - I added the correct gateway address for using t2 so it would work there too.